contact-us/promoted-contacts.html

```diff
@@ -27,30 +27,46 @@ <h1 class="h5">
         <h1 class="h5">
             {{ whistleblowers.title }}
         </h1>
-    {%- if whistleblowers.sitewide_desc != 'none' %}
+    {%- if whistleblowers.sitewide_desc %}
         <p>
             {{ whistleblowers.sitewide_desc }}
         </p>
     {%- endif %}
-    {%- if whistleblowers.email_addr or whistleblowers.phone_num %}
+    {%- if whistleblowers.email or if whistleblowers.phone %}
```

Comment:
Yea but could it be  `{%- if whistleblowers.email or whistleblowers.phone %}`?  Seems more pythonic